index.html

```diff
@@ -721,32 +624,29 @@ <h3>
           <li>Remove |lock| from
           |document|.{{Document/[[ActiveLocks]]}}[|type|].
           </li>
+          <li>[=Queue a task=] on the [=screen wake lock task source=] to:
```

Comment:
The changes to this section also look unrelated to permissions and should be sent in a separate PR.